src/Cameras/Inputs/arcRotateCameraPointersInput.ts

```diff
@@ -144,6 +144,9 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
         var direction = this.pinchInwards ? 1 : -1;
 
         if (this.multiTouchPanAndZoom) {
+            // Zoom and panning enabled together

```

Comment:
Where do you put the comment when it's a `else if` statement in this case?
That's why I put it inside.

Reply:
I'm not 100% that I understand what you're asking but I was just referring to something like below.
```
// Zoom and panning enabled together
if (this.multiTouchPanAndZoom) {
...
```
I originally highlighted this, as a nit because I saw a comment with just whitespace under and thought it looked slightly out of place but wasn't anything to detract from functionality.

Reply:
Sorry, I wasn't clear.

Changing from
``` js
if (this.multiTouchPanAndZoom) {
    // Zoom and panning enabled together
    ...
} else if (this.multiTouchPanning) {
    // Panning enabled, zoom disabled
    ...
}
```
to
``` js
// Zoom and panning enabled together
if (this.multiTouchPanAndZoom) {
    ...
} else if (this.multiTouchPanning) {
    // Panning enabled, zoom disabled    <---- Yuck! Not consistent
    ...
}
```
or
``` js
// Zoom and panning enabled together
if (this.multiTouchPanAndZoom) {
    ...
// Panning enabled, zoom disabled    <---- Mmh why not after all
} else if (this.multiTouchPanning) {
    ...
}
```
seemed weird to me.

But this is definitely just a question of being used to it.
I first thought about option 2 and yuck.
So option 3 ?

Reply:
Yeah, option 3 was what I was suggesting.

Reply:
Ok ! I'm running tests and PR is yours!